### PR TITLE
6.3 Release Notes

### PR DESCRIPTION
**Pull request by jp-tosca**
What this PR does / why we need it:

Dataverse 6.3 release notes.

Which issue(s) this PR closes:

Closes: #10646

### CODE REVIEW

File: doc/release-notes/6.3-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+On forbidden access error page, also know as 403 error page, the math challenge is now correctly display to submit the contact form.
+
+### Ingest Option Bug Fixed
+
+A bug that prevented the Ingest option in the File page Edit File menu from working has been fixed
+
+[↑ Table of Contents](#table-of-contents)
+
+## Persistence
+[↑ Table of Contents](#table-of-contents)
```
Comment (sekmiller) [inline doc/release-notes/6.3-release-notes.md:204]: This is empty and is no longer part of the table of contents above. Remove?

File: doc/release-notes/6.3-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+- [Settings](#settings)
+- [Complete List of Changes](#complete-list-of-changes)
+- [Getting Help](#getting-help)
+- [Upgrade instructions](#upgrade-instructions)
+
+## Release Highlights
+
+### Solr Search and Indexing Improvements
+
+Multiple improvements have been made to the way Solr indexing and searching is done. Response times should be significantly improved.
```
Comment (qqmyers) [inline doc/release-notes/6.3-release-notes.md:21]: "dramatically improved" ! - I don't know what the final results were but things are multiple times faster now - seems like significantly improved isn't enough.

File: doc/release-notes/6.3-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+
+[↑ Table of Contents](#table-of-contents)
+
+## Upgrade Instructions
+
+Upgrading requires a maintenance window and downtime. Please plan accordingly, create backups of your database, etc.
+
+These instructions assume that you've already upgraded through all the 5.x releases and are now running Dataverse 6.2.
+
+0\. These instructions assume that you are upgrading from the immediate previous version. If you are running an earlier version, the only safe way to upgrade is to progress through the upgrades to all the releases in between before attempting the upgrade to this version.
```
Comment (qqmyers) [inline doc/release-notes/6.3-release-notes.md:297]: "only safe way" isn't really true - maybe "only supported way"?

File: doc/release-notes/6.3-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,469 @@
...
+
+Upgrading requires a maintenance window and downtime. Please plan ahead, create backups of your database, etc.
```
Comment (qqmyers) [inline doc/release-notes/6.3-release-notes.md:299]: duplicate

File: doc/release-notes/6.3-release-notes.md (hunk 2 above)
Comment (pdurbin) [reply doc/release-notes/6.3-release-notes.md:21]: I actually like significantly improved. I'd rather underpromise and over deliver. We can always tweak the release note post-release.

File: doc/release-notes/6.3-release-notes.md (hunk 1 above)
Comment (pdurbin) [reply doc/release-notes/6.3-release-notes.md:209]: "Persistence" was removed. Resolving.

File: doc/release-notes/6.3-release-notes.md (hunk 3 above)
Comment (pdurbin) [reply doc/release-notes/6.3-release-notes.md:295]: Sure, see by [Our experience on upgrading dataverse from 4.2.2 to 6.1 in a non-standard way](https://groups.google.com/g/dataverse-community/c/BUO37-reWIs/m/18SK0DZlAQAJ) by @wingfranky as an example of an alternative way of upgrading.

File: doc/release-notes/6.3-release-notes.md (hunk 4 above)
Comment (pdurbin) [reply doc/release-notes/6.3-release-notes.md:297]: Good catch. Will fix.

File: doc/release-notes/6.3-release-notes.md (hunk 3 above)
Comment (pdurbin) [reply doc/release-notes/6.3-release-notes.md]: I changed it to "supported" in 660b72b73d.

File: doc/release-notes/6.3-release-notes.md (hunk 4 above)
Comment (pdurbin) [reply doc/release-notes/6.3-release-notes.md]: Fixed in 660b72b73d

**Review by pdurbin — APPROVED** [review]
I'm happy with the release notes now. Ship it!

We plan to draft a shorter, human-readable version in a Google doc. See also https://iqss.slack.com/archives/C03R1E7T4KA/p1718224575274479

**Pull request merged**